jekyll/_posts/2021-09-25-version-160-released.md

```diff
@@ -0,0 +1,891 @@
+---
+title: "Version 1.6.0 released"
+author: The Nim Team
+---
+
+Nim version 1.6 is now officially released!
+
+A year in the making, 1.6 is the latest stable release and by far the largest yet.
+We're proud of what we, the core team and dedicated volunteers, have accomplished with this milestone:
+* 1667 PRs merged (1760 commits)
+* 893 issues closed
+* 15 new stdlib modules
+* new features in more than 40 stdlib modules, including major improvements to 10 commonly used modules.
+* documentation and minor improvements to 170 modules, including 312 new runnable examples
+* 280 new nimble packages
+
+Nim made its first entry in TIOBE index in 2017 at position 129,
+last year it entered the top-100, and in the last 2 months the top-50 ([https://forum.nim-lang.org/t/8297](https://forum.nim-lang.org/t/8297)).
+We hope this release will reinforce this trend, building on Nim's core strengths:
+a practical, compiled systems programming language; offering C-like performance and portability;
+Python-like syntax; LISP-like flexibility; strong C, C++, JS, Python interop;
+and best-in class metaprogramming.
+
+Why use Nim?
+* one language to rule them all: write your iOS/android/web frontend and backend in the same language, from a [gameboy kickstarter](https://forum.nim-lang.org/t/8375) to a [blockchain client](https://github.com/status-im).
```

Comment:
```suggestion
* one language to rule them all: write your iOS/Android/web frontend and backend in the same language, from a [Gameboy kickstarter](https://forum.nim-lang.org/t/8375) to a [blockchain client](https://github.com/status-im).
```
Does Nim have a working Android backend?

Reply:
It's possible to use nim on android, although better support and adding android to CI would be things we'd need to do; eg:
* https://nim-lang.github.io/Nim/nimc.html#crossminuscompilation-for-android
* https://our.status.im/building-nimbus-on-android/
* https://forum.nim-lang.org/t/5197
and other links